cmd/publishing-bot/healthz.go

```diff
@@ -77,7 +79,18 @@ func (h *Healthz) Run(port int) error {
 	return nil
 }
 
-func (h *Healthz) handler(w http.ResponseWriter, r *http.Request) {
+func (h *Healthz) runHandler(w http.ResponseWriter, r *http.Request) {
+	if h.RunChan == nil {
+		http.Error(w, "run channel is closed", http.StatusInternalServerError)
+		return
+	}
+	h.mutex.RLock()
+	h.RunChan <- true
```

Comment:
No need for the mutex here. We don't change the channel ever.

Reply:
And do:
```
select {
  case h.RunChan <- true:
  default:
}
```

Reply:
@sttts do where?